src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,48 @@
+.calendar {
+  margin: auto;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  gap: $day-gap;
+  max-width: ($day-size * 7) + ($day-gap * 6);
+
+  @each $day, $count in $days {
+    &--start-day-#{$day} :first-child {
+      margin-left: ($day-size + $day-gap) * $count;
+    }
+  }
+
+  @for $i from 28 through 31 {
+    &--month-length-#{$i} :nth-child(n + #{$i + 1}) {
+      display: none;
+    }
+  }
+
+  &__day {
+    background-color: $day-bg-color;
+    width: $day-size;
+    height: $day-size;
+    border: 1px solid black;
```

Comment:
The border color should be defined in the variables file for consistency and maintainability, as per the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules). Consider creating a variable for the border color.